### PR TITLE
Add better error messages for both booking journeys

### PR DESCRIPTION
**Pull request by Guntrisoft**
DWP feedback suggested improved content for error messages.

screen shot 2017-04-10 at 14 01 18

### CODE REVIEW

**Comment by Guntrisoft** [comment]
[Improve error messages for online booking](https://trello.com/c/d9IAARmo/102-improve-error-messages-for-online-booking)

File: app/views/booking_requests/step_two.html.erb [hunk 1]
```diff
@@ -5,13 +5,14 @@
...
-        </h1>
+        <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
+          Unable to submit the form
+        </h2>
+        <p>Check the following:</p>
 
         <ul class="error-summary-list">
-          <% @booking_request.errors.full_messages.each do |msg| %>
-            <li><%= msg %></li>
+          <% @booking_request.errors.each do |key, error_msg| %>
```
Comment (benlovell) [inline app/views/booking_requests/step_two.html.erb]: Can we change the second yielded variable to `message`? It has enough context so doesn't need the `error` prefix and abbreviating variables is a no-no despite it being `msg` before.

File: app/views/telephone_appointments/new.html.erb [hunk 2]
```diff
@@ -1,16 +1,17 @@
...
-        </h1>
+        <h2 class="heading-medium error-summary-heading" id="error-summary-heading-example-1">
+          Unable to submit the form
+        </h2>
+        <p>Check the following:</p>
 
         <ul class="error-summary-list">
-          <% @telephone_appointment.errors.full_messages.each do |msg| %>
-            <li><%= msg %></li>
+          <% @telephone_appointment.errors.each do |key, error_msg| %>
```
Comment (benlovell) [inline app/views/telephone_appointments/new.html.erb]: Same here.

File: app/views/booking_requests/step_two.html.erb (hunk 1 above)
Comment (Guntrisoft) [inline app/views/booking_requests/step_two.html.erb]: Done

**Pull request merged**